docs/concepts-basics/building-blocks/groups.md

```diff
@@ -1,5 +1,38 @@
 # Groups
 
-_Groups_ are made up of users who have roles. An organization can have one or more groups. Each project can be assigned one or more groups. Groups can be assigned to multiple projects. Groups are created independently of projects, and then assigned to them.
+_Groups_ are made up of users who have roles.
+An organization can have one or more groups.
+Each project can be assigned one or more groups.
+Groups can be assigned to multiple projects.
+Groups are created independently of projects, and then assigned to them.
 
-Organizations have a quota to limit the number of groups assigned to it. If you need to change the quota, please contact {{ defaults.helpstring }}.
+Organizations have a quota to limit the number of groups assigned to it.
+If you need to change the quota, please contact {{ defaults.helpstring }}.
+
+# Subgroups
+
+!!! warning
+    Subgroups and not commonly used in Lagoon, and you probably don't need them.
+    However, they can be useful in some situations.
+
+Groups can be nested to an arbitrary depth.
+Subgroups are groups which are a child of another group.
+
+A user's role in an ancestor group is inherited by descendant groups.
+
+## Subgroups example
```

Comment:
```suggestion
## Subgroups explainer
```

Reply:
I'm not sure that "explainer" is clearer than "example" (is it even a word?)

Reply:
We're trying to deter people from creating them - as there's no guarantee that it'll be something Lagoon always supports - ad they introduce a lot of complexity that is better resolved elsewhere. Rather than telling people how to create them, I wanted to explain how they work when inherited (eg via gitlab sync)

Reply:
> there's no guarantee that it'll be something Lagoon always supports - ad they introduce a lot of complexity that is better resolved elsewhere

Should we say that directly in this section then?

Reply:
I updated the title of the section to hopefully be clearer, and also enhanced the warning on the page. Let me know if that is any better?